components/user-store/org.wso2.carbon.identity.user.store.configuration/src/main/java/org/wso2/carbon/identity/user/store/configuration/utils/SecondaryUserStoreConfigurationUtil.java

```diff
@@ -709,4 +707,16 @@ public static void triggerListnersOnUserStorePreDelete(String domainName) throws
                     (), domainName);
         }
     }
+
+    /**
+     * Checks whether having user stores based on separate repositories are supported.
+     *
+     * @return True if repository separation is enabled.
+     */
+    public static boolean isUserStoreRepositorySeparationEnabled() {
+
+        // Support for user stores based on different repositories is
+        // disabled as the feature has on going improvements.
```

Comment:
Better to refer a public git issue from here.

Reply:
Added a reference